webcompat/static/js/lib/models/issue.js

```diff
@@ -91,9 +91,14 @@ issues.Issue = Backbone.Model.extend({
 
   getDescription: function(body) {
     // Get the Description of the body
-    var regex = /<strong>Description<\/strong>: (.+?(?=<br>))/;
-    var description = regex.exec(body);
-    return description != null ? description[1].slice(0, 74) : null;
+    var el = document.createElement("html");
+    el.innerHTML = body;
+    el = el.getElementsByTagName("strong");
+    if (el[5]) {
+      var description = el[5].nextSibling.textContent;
+      description = description.replace(": ", "");
+      return description;
+    }
   },
```

Comment:
Is this more or less what you were expecting @karlcow ?
I had to add the `if` because some mock outs don't have the description attribute and they were breaking the tests.

I don't know how the tests are built... But I don't think we should have so much disparity between tests, it always cause me this kind of problem, and IMO we shouldn't have to add code to the website to deal with things that aren't going to happen in the "real life" for the users.

Reply:
>  I had to add the if because some mock outs don't have the description attribute and they were breaking the tests.

and 

> I don't know how the tests are built... But I don't think we should have so much disparity between tests, it always cause me this kind of problem, and IMO we shouldn't have to add code to the website to deal with things that aren't going to happen in the "real life" for the users.

Agreed but check this :)
https://webcompat.com/issues/1000

we didn't always have a description attribute. 
The description appears with [issue 8073](https://webcompat.com/issues/8073) which was created on 2017-07-07.

Let's [check the changelog](https://github.com/webcompat/webcompat.com/blob/master/CHANGELOG.md#530---2017-07-07). It was changed with PR #1578 fixing issue #1547

Reply:
But now that we have the attribute, shouldn't we include it on the mock outs? So they're more alike with the actual website?

Reply:
If we had unit test cases for the JS functions :) we would indeed cover it. I don't remember we have functional test cases for checking the title. If we have one, we should have the two cases covered. :)